requirements.txt

```diff
@@ -11,7 +11,7 @@ asttokens==2.4.0
 async-lru==2.0.4
 async-timeout==4.0.3
 attrs==23.1.0
-auto-gptq==0.4.2
+# auto-gptq==0.4.2
```

Comment:
I couldn't install this and find any usage in the codebase. Can we remove this from the requirements?

Reply:
I would probably keep this, because, some users utilize quantized model to run KG-RAG. And I presume this was added by them.